bedrock/base/templates/base-new.html

```diff
@@ -0,0 +1,149 @@
+    <meta name="description" content="{% filter striptags|e %}{% block page_desc %}{% endblock %}{% endfilter %}">
+    <meta property="og:type" content="website">
+    <meta property="og:site_name" content="{{ _('Mozilla') }}">
+    <meta property="og:locale" content="{{ LANG|replace("-", "_") }}">
+    <meta property="og:url" content="{% filter trim|absolute_url %}{% block page_og_url %}{{ settings.CANONICAL_URL + '/' + LANG + canonical_path }}{% endblock %}{% endfilter %}">
+    <meta property="og:image" content="{% filter trim|absolute_url %}{% block page_image %}{{ static('img/mozorg/mozilla-256.jpg') }}{% endblock %}{% endfilter %}">
+    <meta property="og:title" content="{% filter striptags|e %}{% block page_og_title %}{{ self.page_title_full() }}{% endblock %}{% endfilter %}">
+    <meta property="og:description" content="{% filter striptags|e %}{% block page_og_desc %}{{ self.page_desc() }}{% endblock %}{% endfilter %}">
+    <meta property="fb:page_id" content="{% block facebook_id %}262134952380{# facebook.com/mozilla #}{% endblock %}">
+    <meta name="twitter:card" content="summary">
+    <meta name="twitter:site" content="@{% block twitter_id %}mozilla{% endblock %}">
+    <meta name="twitter:domain" content="mozilla.org">
+    <meta name="twitter:app:name:googleplay" content="{% block android_app_name %}{{ _('Firefox') }}{% endblock %}">
+    <meta name="twitter:app:id:googleplay" content="{% block android_app_id %}org.mozilla.firefox{% endblock %}">
+    <meta name="twitter:app:name:iphone" content="{% block iphone_app_name %}{{ _('Firefox') }}{% endblock %}">
+    <meta name="twitter:app:id:iphone" content="{% block iphone_app_id %}989804926{% endblock %}">
+    <meta name="twitter:app:name:ipad" content="{% block ipad_app_name %}{{ _('Firefox') }}{% endblock %}">
+    <meta name="twitter:app:id:ipad" content="{% block ipad_app_id %}989804926{% endblock %}">
+    <link rel="author" type="text/plain" href="/humans.txt">
+    <link rel="apple-touch-icon" type="image/png" sizes="180x180" href="{% block page_ios_icon %}{{ static('img/favicon/apple-touch-icon-180x180.png') }}{% endblock %}">
+    <link rel="icon" type="image/png" sizes="196x196" href="{% block page_favicon_large %}{{ static('img/favicon/favicon-196x196.png') }}{% endblock %}">
+    <link rel="shortcut icon" href="{% block page_favicon %}{{ static('img/favicon.ico') }}{% endblock %}">
+    {% block canonical_urls %}{% include 'includes/canonical-url.html' %}{% endblock %}
+    {% endblock shared_meta %}
+
+    {{ l10n_css() }}
+
+    <!--[if lte IE 8]>
+      {# Only needed for IE before v9 #}
+      <script src="{{ static('js/libs/html5shiv.js') }}"></script>
+    <![endif]-->
+
+    <!--[if lte IE 8]>
+      {# Basic styles, only for IE7 and lower #}
```

Comment:
Hmm... yeah... my original intent was to make a new one unrelated to sandstone, but we could probably use sandstone's oldIE and it should be just fine. I just haven't fired up IE8 to try it yet.